Gaijinizer patch/Troops.txt

```diff
@@ -6709,41 +6709,47 @@ Arachne (first time)#MTLed
 
 あたいの攻撃をしのぐなんてね……
 > CONTEXT: Troops/141/pages/2/8/Dialogue/enemy10
-...... Han, isn't it so easy? 
-I'm going to beat my favorite Attack ...#MTLed
+......Hmph, you're quite stubborn aren't you?
+
+To think you'd survive my attack......#Strate
 > END STRING
 
 > BEGIN STRING
 クク……品定めはこのくらいでいいかねぇ…
 
 なかなかの上玉じゃないか…♡
 > CONTEXT: Troops/141/pages/2/12/Dialogue/enemy10
-Kuku...I wonder if the order is so much ...
-Isn't it quite a top ball?#MTLed
+Kuku... I suppose that's enough testing you
+for now...
+
+What a high quality article you are... ♡#StrateLocalization
 > END STRING
 
 > BEGIN STRING
 また会う時を楽しみにしているよ…♡
 
 今度はじっくり、巣の中で弱らせて喰ってやる…
 > CONTEXT: Troops/141/pages/2/16/Dialogue/enemy10
-I look forward to seeing you again...♡
-Now let's take it easy, weaken in the nest ...#MTLed
+I look forward to seeing you again... ♡
+
+This time, I'll carefully weaken you in my nest
+and eat you...#Strate
 > END STRING
 
 > BEGIN STRING
 アルケニー（通常）
 > CONTEXT: Troops/142/name
-Arachne (usually)#MTLed
+Arachne (Normal)#MTLed
 > END STRING
 
 > BEGIN STRING
 まだ、喰うにはちょっと早かったみたいだね…
 
 もう少し、熟すのを待つとするよ…
 > CONTEXT: Troops/142/pages/2/13/Dialogue/enemy10
-Still, it seems to have been a bit early to meet
-... I'll wait a little more to mature ...#MTLed
+I suppose it's still a little too early to eat...
+
+I'll wait for you to ripen a bit more...#Strate
 > END STRING
```

Comment:
"it's" -> "you're", if I'm understanding the context here.

Reply:
I suppose both ways work, I originally interpreted it as something like "It's not time to eat yet".

Reply:
Sounds fine if that's what you want.